docs/contributing/cursorless-in-neovim.md

```diff
@@ -40,7 +40,13 @@ debug mode. To do so you need to run the `workbench.action.debug.selectandstart`
 
 The debug logs are written in `C:\path\to\cursorless\packages\cursorless-neovim\out\nvim_node.log`.
 
-NOTE: This will spawn a standalone nvim instance that is independent of VSCode. Consequently after you're done debugging, you need to close nvim.
+NOTE: This will spawn a standalone nvim instance that is independent of VSCode. Consequently after you're done
+debugging, you need to close nvim.
+
+### Running lua tests
+
+Their are separate cursorless and lua tests. You can run the lua tests by entering the `cursorless.nvim` folder and
+running: `busted --run unit`. These tests currently only work on Linux.
```

Comment:
Should we add a vscode launch config or task for this?

Reply:
Sure, will do as part of this PR